load-test/bin/undo-generate-pupil-load-test-data.sql

```diff
@@ -1,16 +1,14 @@
-
-UPDATE mtc_admin.school SET pin=NULL
-DELETE FROM mtc_admin.psychometricianReportCache
-DELETE FROM mtc_admin.pupilRestart
-DELETE FROM mtc_admin.[checkPin]
-DELETE FROM mtc_admin.answer
+truncate table mtc_admin.psychometricianReportCache
+truncate table mtc_admin.pupilRestart
 UPDATE mtc_admin.pupil SET checkComplete=0, currentCheckId=NULL
-DELETE FROM mtc_admin.[checkConfig]
+truncate table mtc_admin.checkPin
+truncate table mtc_admin.checkConfig
 DELETE FROM mtc_admin.[check]
-DELETE FROM mtc_admin.auditLog
-DELETE FROM mtc_admin.pupilAccessArrangements
-DELETE FROM mtc_admin.pupilAgeReason
```

Comment:
just wondering about this change.
how come `pupilAgeReason` will remain intact on the undo script?

Reply:
its for load testing and those tables have no inbound foreign keys, so we can truncate them, rather than delete them

Reply:
incoming update will render that section commented out (because its not currently part of the scenario and doesnt get populated) but will add pupilAgeReason 👍